msal/application.py

```diff
@@ -633,16 +633,18 @@ def _acquire_token_silent_by_finding_rt_belongs_to_me_or_my_family(
                 **kwargs)
             if at and "error" not in at:
                 return at
+        last_resp = None
         if app_metadata.get("family_id"):  # Meaning this app belongs to this family
-            at = self._acquire_token_silent_by_finding_specific_refresh_token(
+            last_resp = self._acquire_token_silent_by_finding_specific_refresh_token(
```

Comment:
Conceptually, the original `at` name hints it is (mainly) an AT response. Let's keep that semantics. You can use the following trick (not available in most other languages) to have BOTH `at` and `last_resp`.
```suggestion
            last_resp = at = self._acquire_token_silent_by_finding_specific_refresh_token(
```